source/whats-new.txt

```diff
@@ -220,3 +221,31 @@ The 2.22 driver release includes the following new features:
   information on these events, see the
   `MongoDB SDAM Logging and Monitoring specification.
   <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst#log-messages>`__
+
+.. _csharp-previous-versions:
+
+Previous Versions
+-----------------
+
+To learn about the features of previous versions of the {+driver-short+}, visit the
+following links:
+
+- `Version 2.18 <https://mongodb.github.io/mongo-csharp-driver/2.18/>`__
```

Comment:
S: Since this is the What's New page, does it make sense to add a single link for all previous versions, and have it point [here](https://mongodb.github.io/mongo-csharp-driver/2.18/what_is_new/)?